packages/react-graphql/src/async.tsx

```diff
@@ -109,6 +117,7 @@ export function createAsyncQueryComponent<Data, Variables, DeepPartial>({
   FinalComponent.Preload = Preload;
   FinalComponent.Prefetch = Prefetch;
   FinalComponent.KeepFresh = KeepFresh;
+  FinalComponent.resolve = resolve;
```

Comment:
Do we not need a `resolved` key?

Reply:
I am not using it yet. I still believe you that I will need it for test that I am trying to figure out now.